### ISSUE TITLE
Mail> always starts at the first message

### ISSUE THREAD

**Issue by dgets** [dgets/DOCshell#222]
This might have to be clumped in with the issues in #128.  Pointers are not consulted properly, and the _Mail>_ reading process, whether after a posting or just after jumping to _Mail>_, always starts at message 0/1.

**Comment by dgets** [comment]
Trying work on this right now.  Broke a few more things on the way to heaven:

![bbs-mail-autologout-broken](https://cloud.githubusercontent.com/assets/1900617/9151723/eb3a2164-3dc1-11e5-9f53-3c8c6f9e61c1.png)

**Comment by dgets** [comment]
Got another commit going in that rips out a lot of the broken crap that I just inadvertently put in there.  Not everything is back to normal yet, though.  Here's where we're at right now (still got the logout request and all):

![bbs-mail-autologout-broken2](https://cloud.githubusercontent.com/assets/1900617/9151763/94d2b208-3dc3-11e5-977d-c526228db90f.png)

Next commit logged below is of this phase right here.

**Comment by dgets** [comment]
It looks like issues with this were due to attempting to use the `msg_area.sub["mail"]` object in order to determine the last read pointer.  With commit: https://github.com/dgets/DOCshell/commit/98301399627ef55e5673c6f797fb6dde91f079d9 I've managed to regress some of the code to the point where, once again, _Mail>_ functionality is restored (though always starting from 0/1 again).  I'll be consulting **DOVE-Net** to try to figure out what the usual approach is to handling this issue.
